lib/opentracing/span.rb

```diff
@@ -0,0 +1,59 @@
+module OpenTracing
+  # Span represents an OpenTracer Span
+  #
+  # See http://www.opentracing.io for more information.
+  class Span
```

Comment:
same thought about a noop singleton instance (mentioned previously about the SpanContext)